raft/fsm.cc

```diff
@@ -595,11 +595,11 @@ void fsm::tick() {
         // Non leaders will ignore the append reply.
         auto& cfg = get_configuration();
         // If conf is joint it means a leader will send us a non joint one eventually
-        if (!cfg.is_joint() && cfg.current.contains(raft::server_address{_my_id})) {
+        if (!cfg.is_joint() && cfg.current.contains(raft::config_member::from_id(_my_id))) {
```

Comment:
There are containers that support heterogeneous lookup, so you can lookup by `_my_id` directly. Not need for `from_id` functions.

Reply:
Ugh. For `erase`, the generic overload is only available from C++23... -.-